src/app/components/cds/inputs/TextField.js

```diff
@@ -36,6 +36,7 @@ const TextField = React.forwardRef(({
   placeholder,
   required,
   variant,
+  size,
```

Comment:
@sarahkeh can you give some context on adding this size? Since our design system doesn't have different sizes for TextField I hesitate to add this in the codebase. If its to solve an issue on a particular screen, maybe there is a layout change to be made on that screen instead of adding this.

Reply:
It's used on Similarity and translated Statuses page only, just shrinks the height of the input field a little.  I could easily make the change on the screen instead, that way it wouldn't touch the main component.  (also, sorry, didn't notice your second comment until now)

Reply:
@sarahkeh thanks!
If you mean that a place like this (statuses in settings)
<img width="229" alt="Screenshot 2024-04-08 at 8 45 37 PM" src="https://github.com/meedan/check-web/assets/418001/e612feb7-2860-46c2-aeb7-8c40f4b7759a">

That the input is slightly smaller. I would just ignore it completely. I would rather have the CDS textfield input be consistent with the design system and not worry about adding this size attribute. We don't need the maintain the old layout, using the design system more should help everything look consistent.

If when you take size out, if you have a concern about a particular layout, just let me know and we can come up with another solution for those places.

Reply:
Alright, I think the default TextField looks fine, should be good with the base styling, but if you feel differently I can update it.  
![Screenshot 2024-04-15 at 10 38 46 AM](https://github.com/meedan/check-web/assets/161879347/b8b4e59d-5575-41c4-9a1b-cb518263ef21)

Reply:
Thanks @sarahkeh looks good to me. I would just run integration tests again to make sure you got them all fixed up